### PR TITLE
Remove references to SeriesID in `DROP SERIES` handlers.

### LINKED ISSUE #2563: update DROP SERIES to function without series ID but with FROM and WHERE clauses
Current syntax allows for `DROP SERIES <seriesID>`. The only legal syntax for DROP SERIES should be `DROP SERIES FROM database.retentionPolicy.measurement WHERE tagKey = tagValue [AND tag2Key = tag2Value ...]`

### PR DESCRIPTION
**Pull request by toddboom**
Fixes #2563.

### CODE REVIEW

**Comment by corylanou** [comment]
+1 with changelog.

File: influxql/ast.go [hunk 1]
```diff
@@ -1448,9 +1448,6 @@ func (s *ShowSeriesStatement) RequiredPrivileges() ExecutionPrivileges {
 
 // DropSeriesStatement represents a command for removing a series from the database.
 type DropSeriesStatement struct {
-	// The Id of the series being dropped (optional)
-	SeriesID uint64
-
 	// Data source that fields are extracted from (optional)
 	Source Source
```
Comment (dgnorton) [inline influxql/ast.go]: I didn't update this with the regex changes a while back but it probably should be changed to `Sources Sources` and change `parseDropSeriesStatement` to call `parseSources`.  Or, we should add an argument to `parseSource` to disallow regexp.

File: server.go [hunk 2]
```diff
@@ -2568,8 +2556,7 @@ func (s *Server) executeDropSeriesStatement(stmt *influxql.DropSeriesStatement,
 			return nil, ErrDatabaseNotFound(database)
 		}
 
-		// Get the list of measurements we're interested in.
-		measurements, err := measurementsFromSourceOrDB(stmt.Source, db)
+		measurements, err := measurementsFromSourcesOrDB(db, stmt.Sources...)
```
Comment (dgnorton) [inline server.go]: need to call `expandSources` before calling `measurmentsFromSourcesOrDB`

**Comment by dgnorton** [comment]
+1

**Comment by toddboom** [comment]
thanks, guys!

**Pull request merged**